### ISSUE TITLE
PROBLEM with embedded code on Wix Websites

### ISSUE THREAD

**Issue by LucianoNiedermann** [MetaMask/eth-phishing-detect#2023]
Good afternoon,

Whenever I try to add an embedded html code (for a crowdsale token deployment, exchange token deployment,  the CoinLaunch Exchange, etc.) onto my Wix Website it says:

ATTENTION
MetaMask believes this domain to have malicious intent and has prevented you from interacting with it.

This is because the site tested positive on the Ethereum Phishing Detector.

You can turn MetaMask off to interact with this site, but it's advised not to.

How can i fix this problem? I look forward to receiving a solution.

Thank you

Luciano

**Comment by 409H** [comment]
What are you embedding?

**Comment by 409H** [comment]
Can you give the embed codes?

**Comment by cbosco** [comment]
The Spotify sample play button embed code triggers the warning from:

https://developer.spotify.com/documentation/widgets/generate/play-button/

```
<iframe src="https://open.spotify.com/embed/album/1DFixLWuPkv3KT3TnV35m3" width="300" height="380" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>
```

**Comment by 409H** [comment]
Interesting. I cannot replicate it - are you able to give me the link to a WIX site having the issue - maybe WIX is putting it through a proxy?

**Comment by cbosco** [comment]
Thanks @409H I was able to do this using the editor on a new account, but unable to see it in the output (although the output 404s -- apologies I am not too familiar with Wix)

Steps to reproduce:

1. Create free account
2. choose a template (I chose "Coworking")
3. Add widget > More > iFrame

![image](https://user-images.githubusercontent.com/308032/46290804-d79e3b80-c55a-11e8-85eb-78d31bf96fb3.png)

4. Edit code button
![image](https://user-images.githubusercontent.com/308032/46290836-e8e74800-c55a-11e8-8774-6042b0803cff.png)

5. Paste in the markup
![image](https://user-images.githubusercontent.com/308032/46290846-eedd2900-c55a-11e8-92f4-87dfc2bd41db.png)
